### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by abcdesign1**

### CODE REVIEW

File: src/inverseRobot.js [hunk 1]
```diff
@@ -6,8 +6,22 @@
...
+    // console.log('key =', key, ', robot[key] =', robot[key]);
+    if (!tobor[robot[key]]) {
+      tobor[robot[key]] = key;
+    }
+  });
 
+  return Object.keys(robot).length === Object.keys(tobor).length ? tobor : null;
+}
+// console.log(inverseRobot(kolli));
+// console.log(inverseRobot(robert));
```
Comment (illiarerun) [inline src/inverseRobot.js:26]: and this

File: src/inverseRobot.js [hunk 2]
```diff
@@ -6,8 +6,22 @@
...
+// const robert = { Robert: 'name', 123: 'chipVer', 113: 'chipVer' };
+// inverseRobot(robert) === null
+// inverseRobot(kolli) === { name: 'Kolli', chipVer: '123', wheels: '3' }
 function inverseRobot(robot) {
-  // write code here
-}
+  const tobor = {};
+
+  Object.keys(robot).forEach(key => {
+    // console.log('key =', key, ', robot[key] =', robot[key]);
```
Comment (illiarerun) [inline src/inverseRobot.js:17]: and this

File: src/inverseRobot.js [hunk 3]
```diff
@@ -6,8 +6,22 @@
...
+    if (!tobor[robot[key]]) {
+      tobor[robot[key]] = key;
+    }
```
Comment (illiarerun) [inline src/inverseRobot.js:20]: you repeat robot[key] 2 times here, better create variable with this value

**Review by illiarerun — CHANGES_REQUESTED** [review]

File: src/inverseRobot.js [hunk 4]
```diff
@@ -6,8 +6,22 @@
  * @return {object}
 */
 
+// const kolli = { Kolli: 'name', 123: 'chipVer', 3: 'wheels' };
+// const robert = { Robert: 'name', 123: 'chipVer', 113: 'chipVer' };
+// inverseRobot(robert) === null
+// inverseRobot(kolli) === { name: 'Kolli', chipVer: '123', wheels: '3' }
```
[the remark this answers is not in the document]
Comment (vadym4che) [reply src/inverseRobot.js]: removed

File: src/inverseRobot.js (hunk 1 above)
Comment (vadym4che) [reply src/inverseRobot.js]: removed

File: src/inverseRobot.js (hunk 2 above)
Comment (vadym4che) [reply src/inverseRobot.js]: removed

File: src/inverseRobot.js (hunk 3 above)
Comment (vadym4che) [reply src/inverseRobot.js]: const created

**Review by sergii-nosachenko — APPROVED** [review]

**Pull request closed without merging**